### DISCUSSION TITLE
Link GitBook Change Requests to GitHub PRs/Issues and JIRA tickets

### DISCUSSION THREAD

**Discussion by emmerich** [GitbookIO/community#72] [Feature Requests]
### What's your feature/idea?

We often create JIRA tickets (as well as GitHub Issues) for changes we make in the product. These changes usually come with documentation changes too. One or two PRs gets open in GitHub making the product changes and the docs changes. Essentially:

1. a ticket is created in JIRA/GitHub "we need to do x"
2. this work involves documentation changes
3. an engineer picks up the ticket and starts working on both the code
4. the engineer (or someone else in the team picks up the "docs" side of the ticket)
5. they either create a PR in GitHub with the docs change, or create a CR directly in GitBook with the changes

### What's your desired outcome?

We want to be able to link the CRs in GitBook with the tickets/PRs so that we can maintain the link between the 3 tools and potentially automatically close issues when the CR is merged/deployed. 

### What's the impact of this for you?

- [ ] Critical
- [ ] High
- [ ] Medium
- [ ] Low

**Comment by MantisClone** [comment]
Github Issues and Jira Tickets both integrate with Github PRs already. So I think the best way to maintain the link between the 3 tools would be to create a feature that will "push" a CR to Github, effectively mirroring it as a PR.

I'm strongly in favor of this feature because it would allow me to review CRs using Github's PR review tools (which I find to be far easier to use than Gitbooks CR review tools).

Ideally, the PR would update automatically any time the CR is updated, but this seems advanced, so as a first step, it would be acceptable to have a manual sync button that would re-push the CR and update the PR.

Ideally, there would be a way to pull any changes into the CR any time the PR is updated, but this seems advanced because the CR would need to gracefully handle invalid markdown. So as a first step, it would be acceptable to disallow PR to CR sync and build it as a version 2, after proving demand for the workflow.

**Comment by jquentin-lion** [comment]
Absolutely. A change request is basically a Pull Request. Why not connect the 2? That should work both ways: if a Pull Request is created in the repo, that should show up as a Change Request in gitbook?
